### PR TITLE
init project

### PR DESCRIPTION
**Pull request by puteriffah**

### CODE REVIEW

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
```
Comment (manhnt-0960) [inline Gemfile]: use `"` instead of `'`
remove all comment in this file

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: use `"` instead of `'`

File: Gemfile (hunk 1 above)
Comment (puteriffah) [reply Gemfile]: done

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (puteriffah) [reply app/views/layouts/application.html.erb]: done

**Comment by puteriffah** [comment]
ready

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+
+gem 'sqlite3', '~> 1.4'
+
+gem 'puma', '~> 4.1'
```
Comment (ngocvt-0484) [inline Gemfile]: please change ' to "
in rails file, we will use "

File: config/locales/en.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+#     I18n.t 'hello'
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
```
Comment (ngocvt-0484) [inline config/locales/en.yml]: remove all comment

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
```
Comment (ngocvt-0484) [inline config/database.yml]: this file will not be up to GIt, please
cp database.yml database.yml.example
git rm --cached config/database.yml
and in .gitignore add
/config/database.yml

File: config/credentials.yml.enc [hunk 6]
```diff
@@ -0,0 +1 @@
+4mDHKR/uEUO/hUfqo+KzxGHHihYuo89NTBgYfiG7D3+0wVKxZw0KpLAPcj6ZbE8s7mCHX8rruykYmiIkrVt9tux5Rj1iMCVKk24lMTZyvyj1eWsOxq6QqnX4PTe4l2weWQvv9tbQj1pY4UdvJLTaX6Q/gH1VvK7MhrSPdQo8Q4ceVKpoccv4ySL0hX7vYLJpVWg/8t68RJD6yx1UZniuddhwcNdcJrNEl71q6ws6tUTVnqbvbeDkluFu4ke1XublYmKztgcYGD9/uq1//tE1yxNxOtWo9tGrECaZhd4xDp09reL8y5zXPph9IQYlNs90Xma0YnN8AnsWPWXhcNXEqg5lfGkFqJgYfcYm/bUTBACn7goBfCT7jzXkBdh5DZd2CBO3KLzsOR7u/ZODxbQ/yzAxpDWPPEejvXUZ--tAhzQZJcsZhp0hZ1--vxUh096icWMEwTmb8/AcaA==
```
Comment (ngocvt-0484) [inline config/credentials.yml.enc]: same as database.yml

File: config/database.yml (hunk 5 above)
Comment (puteriffah) [reply config/database.yml]: how to add into .gitignore?

Comment (puteriffah) [reply config/database.yml]: git add config/database.yml?

Comment (vanvtt-0952) [reply config/database.yml]: open `.gitgnore` file, and write text `config/database.yml`

File: Gemfile (hunk 1 above)
Comment (vanvtt-0952) [reply Gemfile]: I don't see your changes.

File: app/mailers/application_mailer.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```
Comment (vanvtt-0952) [inline app/mailers/application_mailer.rb]: ruby file, use double-quotes

File: app/javascript/packs/application.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
```
Comment (vanvtt-0952) [inline app/javascript/packs/application.js]: css file and js file, use `single quote`

File: app/mailers/application_mailer.rb (hunk 7 above)
Comment (puteriffah) [reply app/mailers/application_mailer.rb]: i have changed ' to " yesterday but it turnoed out the same.  i will change again

**Comment by puteriffah** [comment]
ready

File: app/javascript/packs/application.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: use ' instead of " for js and css file

File: config/database.yml (hunk 5 above)
Comment (saobv-2223) [reply config/database.yml]: open `.gitignore` and add `config/database.yml`
then run command `git rm --cached config/database.yml`

File: config/credentials.yml.enc (hunk 6 above)
Comment (saobv-2223) [inline config/credentials.yml.enc]: do same as database.yml file

File: test/application_system_test_case.rb [hunk 10]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: We don't use test folder, so you add `test` into `.gitignore` and run `git rm --cached -r test`

**Comment by puteriffah** [comment]
ready

File: app/mailers/application_mailer.rb [hunk 11]
```diff
@@ -0,0 +1,4 @@
...
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: You have not yet fixed them

File: config/locales/en.yml [hunk 12]
```diff
@@ -0,0 +1,33 @@
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
```
Comment (saobv-2223) [inline config/locales/en.yml]: remove all comments

**Comment by puteriffah** [comment]
ready

File: app/mailers/application_mailer.rb (hunk 11 above)
Comment (puteriffah) [reply app/mailers/application_mailer.rb]: i have fixed it again
image

File: Gemfile [hunk 13]
```diff
@@ -0,0 +1,48 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.0"
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+
+gem "sqlite3", "~> 1.4"
```
Comment (ngocvt-0484) [inline Gemfile]: remove this gem
add gem "mysql2"
after that, you bundle install and restart server

Comment (puteriffah) [reply Gemfile]: remove both gem?

**Comment by puteriffah** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**